planet/cli/data.py

```diff
@@ -309,13 +310,18 @@ async def search(ctx, item_types, filter, limit, name, sort, pretty):
     parameter will be applied to the stored quick search.
     """
     async with data_client(ctx) as cl:
-
+        item_ids = []
         async for item in cl.search(item_types,
                                     search_filter=filter,
                                     name=name,
                                     sort=sort,
                                     limit=limit):
-            echo_json(item, pretty)
+            if ids_only:
+                item_ids.append(item['id'])
+            else:
+                echo_json(item, pretty)
```

Comment:
@kevinlacaille In #645 I'm pushing back on this feature request, but if we did it, this is completely sound. Append to list oe print to terminal, this is easy to read and understand. It's symmetry in action.